### DISCUSSION TITLE
[Idea] Custom Tournament Banners

### DISCUSSION THREAD

**Discussion by paranormalraccoon** [etournity/etournity#71] [Ideas]
### 1. Describe your idea

- Tournament Organizers like customization to set themselves apart from the masses and establish their own branding
- we should allow them to use custom banners on their Info Page

### 2. Why / How does this idea improve Etournity?

- Organizer happiness grows

### 3. User Group

- [X] Tournament Organizer
- [ ] Player
- [ ] Tournament Moderator
- [ ] Spectator
- [ ] Other

**Comment by DavidAyliffe** [comment]
This would be great!
